frontend/src/views/Collection/index.tsx

```diff
@@ -91,49 +98,54 @@ const Collection: FC<Props> = ({ id = "" }) => {
   );
 
   return (
-    <ViewGrid>
-      <CollectionInfo>
-        <H3>{collection.name}</H3>
-        <Description>{collection.description}</Description>
-        <LinkContainer>{renderLinks(collection.links)}</LinkContainer>
-      </CollectionInfo>
-
-      <CollectionButtons>
-        {collection.access_type === ACCESS_TYPE.WRITE && isPrivate && (
-          <DeleteCollection id={id} />
-        )}
-        <Button
-          intent={Intent.PRIMARY}
-          minimal
-          outlined
-          disabled
-          text="Share"
-        />
-        {isPrivate && (
-          <PublishCollection isPublishable={isPublishable} id={id} />
-        )}
-      </CollectionButtons>
-      <DatasetContainer>
-        {isDatasetPresent ? (
-          <DatasetsGrid
-            datasets={datasets}
-            uploadedFiles={uploadedFiles}
-            invalidateCollectionQuery={invalidateCollectionQuery}
-            onSelect={setSelected}
+    <>
```

Comment:
The only change here is adding a wrapper `<></>` and a `<Head />` sibling to `<ViewGrid />`